CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### 2.0.0
 
+- Added support for [UPLOADCARE_STORE](https://uploadcare.com/documentation/upload/#upload-body) flag
```

Comment:
`Added support for `store` flag in [Upload API](https://uploadcare.com/documentation/upload/) methods.`